src/libexpr/primops/fetchTree.cc

```diff
@@ -33,9 +33,8 @@ void emitTreeAttrs(
 
     // FIXME: support arbitrary input attributes.
 
-    auto narHash = input.getNarHash();
-    assert(narHash);
-    attrs.alloc("narHash").mkString(narHash->to_string(HashFormat::SRI, true));
+    if (auto narHash = input.getNarHash())
```

Comment:
While probably not an issue, I don't think you intended to include this in this PR considering there's another PR open for it: https://github.com/NixOS/nix/pull/11933

Reply:
Oops, indeed.